sympy/integrals/tests/test_deltafunctions.py

```diff
@@ -35,8 +35,9 @@ def test_deltaintegrate():
     assert deltaintegrate(DiracDelta(x)**2, x) == DiracDelta(0)*Heaviside(x)
     assert deltaintegrate(y*DiracDelta(x)**2, x) == \
         y*DiracDelta(0)*Heaviside(x)
-    assert deltaintegrate(DiracDelta(x, 1)**2, x) is None
-    assert deltaintegrate(y*DiracDelta(x, 1)**2, x) is None
+    assert deltaintegrate(DiracDelta(x, 1)**2, x) == -DiracDelta(0, 2)*Heaviside(x)
```

Comment:
Why is the delta squared in these tests? We should also add tests for the delta term not squared (like the ones shown in the issues this PR is marked as fixing).